api/server/init.go

```diff
@@ -56,6 +58,19 @@ func getEnvInt(key string, fallback int) int {
 	return fallback
 }
 
+func getEnvDuration(key string, fallback time.Duration) time.Duration {
+	var err error
+	res := fallback
+	if tmp := os.Getenv(key); tmp != "" {
+		res, err = time.ParseDuration(tmp + "s")
+		if err != nil {
+			err = fmt.Errorf("%s invalid %s=%s", err.Error(), key, tmp)
```

Comment:
I'm not sure I'm following this format string. The error is invalid?

I would probably use `logrus.WithFields().Fatal()` here, but that would be slightly different to the `getEnvInt` behaviour.

Reply:
It would give a message something like this - panic: time: invalid duration abcd invalid ENV_VAR=abcd if we had passed abcd as the duration be parsed.. just to make the panic message more understandable